test/onigumo_test.exs

```diff
@@ -50,11 +35,12 @@ defmodule OnigumoTest do
     input_file_content = Enum.map(@urls, &(&1 <> "\n")) |> Enum.join()
     File.write!(input_path_tmp, input_file_content)
 
-    download_result = Onigumo.download(@urls, HTTPoisonMock, tmp_dir)
-    expected_responses = Enum.map(@urls, fn _ -> :ok end)
-    assert(download_result == expected_responses)
-
     output_path = Path.join(tmp_dir, @output_path)
+    Enum.map(@urls, fn url ->
+      download_result = Onigumo.download(url, HTTPoisonMock, output_path)
+      assert(download_result == :ok)
+    end)
+
```

Comment:
The test still does not do what it should. Absence of `download` for a list of URLs only exposes this more.

Reply:
Now, after [merging](https://github.com/Glutexo/onigumo/pull/67/commits/278d1e0a7b666c573b0912ab28bcb6c3c22a565c) current [_elixir_](https://github.com/Glutexo/onigumo/tree/elixir) containing #68, the tests works as originally intended without any further changes. 👍🏻